.circleci/config.yml

```diff
@@ -59,8 +59,38 @@ phpunit: &phpunit
       # run tests!
       - run: vendor/bin/phpunit
 
+phpunit1: &phpunit1
```

Comment:
Let's rename this to `phpunit-lowest`, and then feel free to merge if the jobs all pass. Nice work!

Reply:
Thanks @deviantintegral 👍  Yup ill rename it and merge when all the jobs pass.